### PR TITLE
fix log output of collect[] params

### PR DESCRIPTION
**Pull request by ishii1648**
The following codes always output 'msg="collect[] params" params="unsupported value type"'.
```
level.Debug(logger).Log("msg", "collect[] params", "params", params)
```

This "params" variable is slice, so should be split to string.

This is example log output by modified code.
```
msg="collect[] params" params=info_schema.innodb_cmp+nfo_schema.query_response_time
```

### CODE REVIEW

File: mysqld_exporter.go [hunk 1]
```diff
@@ -195,7 +196,7 @@ func newHandler(metrics collector.Metrics, scrapers []collector.Scraper, logger
 				r = r.WithContext(ctx)
 			}
 		}
-		level.Debug(logger).Log("msg", "collect[] params", "params", params)
+		level.Debug(logger).Log("msg", "collect[] params", "params", strings.Join(params, "+"))
```
Comment (SuperQ) [inline mysqld_exporter.go:199]: I think a comma would be better.

```suggestion
		level.Debug(logger).Log("msg", "collect[] params", "params", strings.Join(params, ","))
```

**Comment by SuperQ** [comment]
Thanks, minor change request.

File: mysqld_exporter.go (hunk 1 above)
Comment (ishii1648) [reply mysqld_exporter.go:199]: Thanks. I got it!

**Review by SuperQ — APPROVED** [review]
LGTM

**Pull request merged**